cohere/client.py

```diff
@@ -284,6 +295,16 @@ def _validate_chat_history(self, chat_history: List[Dict[str, str]]) -> None:
             if not isinstance(entry["user_name"], str) or not isinstance(entry["text"], str):
                 raise CohereError(message="both user_name and text must be strings in chat_history.")
 
+    def _validate_chat_params(self, logit_bias: Dict[int, float]) -> None:
```

Comment:
I think we just leave this up to the API. also a float,int check can easily fail with numpy types or such, and you end up with rather complicated checks (e.g. [numbers.Number](https://docs.python.org/library/numbers.html#numbers.Number))